starter_code/lab-express-cinema/views/movie-info.hbs

```diff
@@ -0,0 +1,6 @@
+<h2>{{movieItem.title}}</h2>
+<p>{{movieItem.director}}</p>
+<p>{{movieItem.description}}</p>
+<p> {{movieItem.showtime}}</p>
```

Comment:
Ah, actually small thing to be careful here : showtimes is an array, to display the showtimes you thus have to loop over it : 
```
<ul>
  {{#each movieItem.showtimes}}

  <li> {{this}}</li>


  {{/each}}
</ul>
```